src/syntax-tree/ast.ts

```diff
@@ -1627,84 +1589,84 @@ export class ListLiteralExpression extends Expression {
         this.indexInRoot = indexInRoot;
 
         this.tokens.push(new NonEditableTkn("[", this, this.tokens.length));
-        this.tokens.push(new EmptyListItem(true, this, this.tokens.length));
+        this.tokens.push(new EmptyExpr(this, this.tokens.length));
```

Comment:
Should we use TypedEmptyExpr with type Any since we are going to be removing EmptyExpr eventually? It would help me find all places where Any needs to be supported as well instead of discovering them later.

Reply:
Sure. Will do that.